cmd/litcli/ln.go

```diff
@@ -223,8 +224,61 @@ var (
 		Usage: "the asset ID of the asset to use when sending " +
 			"payments with assets",
 	}
+
+	assetAmountFlag = cli.Uint64Flag{
+		Name: "asset_amount",
+		Usage: "the amount of the asset to send in the asset keysend " +
+			"payment",
+	}
+
+	rfqPeerPubKeyFlag = cli.StringFlag{
+		Name: "rfq_peer_pubkey",
+		Usage: "(optional) the public key of the peer to ask for a " +
+			"quote when converting from assets to sats; must be " +
+			"set if there are multiple channels with the same " +
+			"asset ID present",
```

Comment:
Currently, this must be set before it reaches the tap RFQ service. If this field is optional, and unset by the user, when does it get populated?

Reply:
Tapd is able to find a peer pubkey, given a specific asset id (it will look for a peer that has that asset channel)